mne/io/kit/kit.py

```diff
@@ -563,160 +544,214 @@ def get_kit_info(rawfile):
-                                      "contact the MNE-Python developers."
-                                      % (sysname, sysid))
-        KIT_SYS = KIT_CONSTANTS[sysid]
-        logger.info("KIT-System ID %i: %s" % (sysid, sysname))
-        if sysid in SYSNAMES:
-            if sysname != SYSNAMES[sysid]:
-                warn("KIT file %s has system-name %r, expected %r"
-                     % (rawfile, sysname, SYSNAMES[sysid]))
-
+        system_name = unpack('128s', fid.read(128))[0].decode()
+        # use first line as shorthand
+        sysname = system_name[:system_name.index('\n')]
+        # model name
+        model_name = unpack('128s', fid.read(128))[0].decode()
         # channels
-        fid.seek(KIT.STRING, SEEK_CUR)  # skips modelname
-        sqd['nchan'] = unpack('i', fid.read(KIT.INT))[0]
-        # channel locations
-        fid.seek(KIT_SYS.CHAN_LOC_OFFSET)
-        chan_offset = unpack('i', fid.read(KIT.INT))[0]
-        chan_size = unpack('i', fid.read(KIT.INT))[0]
-
-        fid.seek(chan_offset)
-        sensors = []
-        for i in range(KIT_SYS.N_SENS):
+        sqd['nchan'] = channel_count = unpack('i', fid.read(KIT.INT))[0]
+        comment = unpack('256s', fid.read(256))[0].decode()
+        create_time, last_modified_time = unpack('2i', fid.read(2 * KIT.INT))
+        fid.seek(KIT.INT * 3, SEEK_CUR)  # reserved
+        dewar_style = unpack('i', fid.read(KIT.INT))[0]
+        fid.seek(KIT.INT * 3, SEEK_CUR)  # spare
+        fll_type = unpack('i', fid.read(KIT.INT))[0]
+        fid.seek(KIT.INT * 3, SEEK_CUR)  # spare
+        trigger_type = unpack('i', fid.read(KIT.INT))[0]
+        fid.seek(KIT.INT * 3, SEEK_CUR)  # spare
+        adboard_type = unpack('i', fid.read(KIT.INT))[0]
+        fid.seek(KIT.INT * 29, SEEK_CUR)  # reserved
+
+        if version == 2 and revision == 3:
+            adc_range = float(unpack('i', fid.read(KIT.INT))[0])
+        else:
+            adc_range = unpack('d', fid.read(KIT.DOUBLE))[0]
+        adc_polarity, adc_allocated, adc_stored = unpack('3i',
+                                                         fid.read(3 * KIT.INT))
+
+        logger.info("SQD file basic information:")
+        logger.info("Meg160 version = V%iR%03i", version, revision)
+        logger.info("System ID      = %i", sysid)
+        logger.info("System name    = %s", system_name.replace('\n', '/'))
+        logger.info("Model name     = %s", model_name)
+        logger.info("Channel count  = %i", channel_count)
+        logger.info("Comment        = %s", comment)
+        logger.info("Dewar style    = %i", dewar_style)
+        logger.info("FLL type       = %i", fll_type)
+        logger.info("Trigger type   = %i", trigger_type)
+        logger.info("A/D board type = %i", adboard_type)
+        logger.info("ADC range      = +/-%s[V]", adc_range / 2.)
+        logger.info("ADC allocate   = %i[bit]", adc_allocated)
+        logger.info("ADC bit        = %i[bit]", adc_stored)
```

Comment:
Some or all of these should probably be `logger.debug`. Maybe just the MEG version, system, ID, name, and model name?